clients/da-vinci-client/src/test/java/com/linkedin/davinci/kafka/consumer/StoreIngestionTaskTest.java

```diff
@@ -3978,11 +3984,11 @@ private Consumer<PubSubTopicPartitionOffset> getObserver(
 
   @Test
   public void testResubscribeAfterRoleChange() throws Exception {
-    PubSubTopic realTimeTopic =
-        pubSubTopicRepository.getTopic(Version.composeRealTimeTopic(storeNameWithoutVersionInfo));
+    String realTimeTopicName = storeNameWithoutVersionInfo + Version.REAL_TIME_TOPIC_SUFFIX;
```

Comment:
Same here: use `Utils.composeRealTimeTopic`

Reply:
Also we don't really need to create var: realTimeTopicName

Reply:
I wanted to avoid duplicate `Version.composeRealTimeTopic(storeNameWithoutVersionInfo)`